src/utils/embedding_store.py

```
@@ -0,0 +1,425 @@
+                        created_at=message_created_at,
+                    )
+                    document_objects.append(doc)
+
+                # Batch insert all documents
+                db.add_all(document_objects)
+                await db.commit()
+                logger.debug(
+                    f"Batch created {len(document_objects)} unified observations"
+                )
+
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)